libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -526,7 +526,8 @@ void AP_Airspeed::calibrate(bool in_startup)
             continue;
         }
         if (sensor[i] == nullptr) {
-            GCS_SEND_TEXT(MAV_SEVERITY_ERROR, "Airspeed %u not initalized, cannot cal", i+1);
+            GCS_SEND_TEXT(MAV_SEVERITY_ERROR, "Airspeed %u not initialized, cannot calibrate", i+1);
+            calibration_state[i] = CalibrationState::FAILED;
```

Comment:
we could only set FAILED state if the sensor is setup to be used, as otherwise we will get a lot of failed reports with the default setup due to default ARSPD_TYPE as @peterbarker points out

Reply:
That's a reasonable middle ground, I'll rework with that added.